src/xref2/subst.ml

```diff
@@ -862,15 +875,24 @@ and rename_bound_idents s sg =
       in
       rename_bound_idents s
         ( Include
-            { i with Component.Include.expansion_ = { items; removed = [] } }
+            {
+              i with
+              Component.Include.expansion_ =
+                { items; removed = []; compiled = false };
```

Comment:
Shouldn't be copying `compiled` ? (this probably doesn't matter here because it's handled in `apply_sig_map`)

Reply:
Yes I think you're right - we should preserve it here, then we can override it with the logic in `apply_sig_map` later.